docs/admin/authentication.md

```diff
@@ -1,4 +1,10 @@
 ---
+assignees:
+- david-mcmahon
```

Comment:
did david-mcmahon make a big swath of changes? i see his name on many of these files